tests/ci/install.sh

```diff
@@ -62,6 +62,15 @@ if [ -n "${ACMS_JOB}" ]; then
     composer config --unset repositories.acquia_cms_common
     composer require drupal/acquia_cms_common:2.x-dev -W
   fi
+  if [ "${ACMS_JOB}" == "dev_version_test" ]; then
+    composer config extra.composer-exit-on-patch-failure false
```

Comment:
@rajeshreeputra This adds value as string, please use this instead
```suggestion
    composer config extra.composer-exit-on-patch-failure "false" --json 
```